packages/ts-moose-lib/src/moose-runner.ts

```diff
@@ -10,6 +10,8 @@ register({
   experimentalTsImportSpecifiers: true,
 });
 
+import "./instrumentation";
```

Comment:
i think it looks nicer if we call a function from the file instead of importing for side effects

Reply:
That actually doesn't work - I just tried